Dockerfile

```
@@ -1,17 +1,16 @@
-FROM quay.io/giantswarm/alpine:3.12 AS binaries
+FROM quay.io/giantswarm/alpine:3.14 AS binaries
 
-ARG KUBECTL_VERSION=1.18.9
+ARG KUBECTL_VERSION=1.29.2
 
 RUN apk add --no-cache ca-certificates curl \
     && mkdir -p /binaries \
     && curl -SL https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /binaries/kubectl \
```

add `-f` flag to make curl exit non-zero if the response was not successful

```suggestion
    && curl -SLf https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /binaries/kubectl \
```